code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -76,8 +76,10 @@
 	victims -= mind.current
 	if(!victims.len)
 		return
-	var/mob/living/carbon/T = input(src, "Victim?") as null|anything in victims
-
+	var/mob/living/carbon/T 
```

Comment:
for future reference, you can make this line `var/mob/living/carbon/T = victims[1]`